packages/wallets/src/features/cashier/flows/WalletWithdrawal/WalletWithdrawal.tsx

```diff
@@ -41,33 +35,21 @@ const WalletWithdrawal = () => {
 
     if (verificationCode) {
         if (isCurrencyConfigSuccess && activeWallet?.currency) {
-            return (
-                <CashierLocked module='withdrawal'>
-                    <WithdrawalLocked>
-                        {isCrypto ? (
-                            <WithdrawalCryptoModule
-                                onClose={() => {
-                                    setVerificationCode('');
-                                }}
-                                verificationCode={verificationCode}
-                            />
-                        ) : (
-                            <WithdrawalFiatModule verificationCode={verificationCode} />
-                        )}
-                    </WithdrawalLocked>
-                </CashierLocked>
+            return isCrypto ? (
+                <WithdrawalCryptoModule
+                    onClose={() => {
+                        setVerificationCode('');
+                    }}
+                    verificationCode={verificationCode}
+                />
+            ) : (
+                <WithdrawalFiatModule verificationCode={verificationCode} />
```

Comment:
[Same](https://github.com/binary-com/deriv-app/pull/14585/files#r1565538535) here

Reply:
hey @aum-deriv 
actually i don't see any issue here to lift up locked scenarios to the parent.
conversely, when we lift up locked scenarios we eliminate all the logic we are running in WalletWithdrawal component, we avoid to call useEffect, useCurrencyConfig, useAuthorize, useActiveWalletAccount...
The same thing with other flows, Deposit, Transfer, etc